src/components/UI/Modal/Modal/Modal.js

```diff
@@ -31,17 +31,23 @@ export const Modal = ({
   show,
   type = ModalType.INFO,
   size = ModalSize.MEDIUM,
+  children,
   containerStyle,
-  children
+  exitable,
+  hideModal
 }) => {
   const {width} = size;
 
   return show
     ? createPortal(
-        <div className={toClasses(styles.modal, styles[type])}>
+        <div
+          className={toClasses(styles.modal, styles[type])}
+          onClick={() => exitable && hideModal()}
```

Comment:
Export to named function (anonymous functions as event handlers are bad practice).